chromium_src/extensions/common/permissions/api_permission.h

```diff
@@ -0,0 +1,15 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_CHROMIUM_SRC_EXTENSIONS_COMMON_PERMISSIONS_API_PERMISSION_H_
+#define BRAVE_CHROMIUM_SRC_EXTENSIONS_COMMON_PERMISSIONS_API_PERMISSION_H_
+
+#define BRAVE_ID kIpfs = 231,

```

Comment:
Maybe call this BRAVE_PERMISSION_IDS with an S because it signifies how we'll patch new permissions in the future, by adding more IDs here.

Reply:
done